.github/workflows/weekly_blog_and_website_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Weekly blog and website report
+
+on:
+  schedule:
+    - cron: '0 4 * * WED'  # Run at 04:00 UTC every Wednesday
+  workflow_dispatch:
+
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+    if: github.repository == 'leanprover-community/mathlib4'
+
+    steps:
+    - name: Checkout website
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        repository: 'leanprover-community/leanprover-community.github.io'
+        path: website
+
+    - name: Checkout blog
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        repository: 'leanprover-community/leanprover-community.github.io'
+        path: blog
+
+    - name: Produce tables
+      id: tables
+      run: |
+        for site in website blog
+        do
+          printf 'PRs in %s\n' "${site}"
+          cd "${site}"
+          printf $'%s<<EOF\n%s\nEOF' "${site}" "$(./formatGhPrList.sh "${site}")" | tee -a "$GITHUB_OUTPUT"
```

Comment:
I think we need to checkout the azure-scripts repo too for this to work.

Reply:
Good catch: done!